src/action/channel.js

```diff
@@ -259,13 +259,24 @@ class ChannelAction {
    * the necessary error handling and notification display.
    * @return {Promise<undefined>}
    */
-  async closeSelectedChannel() {
+  async closeSelectedChannel(force = false) {
     try {
       const { selectedChannel } = this._store;
       this._nav.goChannels();
-      await this.closeChannel({ channelPoint: selectedChannel.channelPoint });
+      await this.closeChannel({
+        channelPoint: selectedChannel.channelPoint,
+        force,
+      });
     } catch (err) {
-      this._notification.display({ msg: 'Closing channel failed!', err });
+      if (
+        err &&
+        err.details &&
+        err.details.includes('try force closing it instead')
```

Comment:
We're checking the string contents of the error message here because LND returns a generic/unknown error code.

https://github.com/lightningnetwork/lnd/blob/master/rpcserver.go#L1138
https://github.com/grpc/grpc-go/blob/master/codes/codes.go#L43